app/frontend/src/cljs/teet/common/common_controller.cljs

```diff
@@ -308,26 +308,27 @@
                                                      {:error :request-timeout})))
                                   timeout-ms)))]))
 
-(def ^:private fetch-timeout-ms 30000)
+(def ^:private default-fetch-timeout-ms 30000)
```

Comment:
The changes in this ns add the option to provide a custom timeout for queries, commands etc.